From 638e8e4c6c200c9d0d49ad9f0771dc6a6119f01c Mon Sep 17 00:00:00 2001
From: Stephan Witt <switt@lyx.org>
Date: Wed, 9 Feb 2022 10:59:18 +0100
Subject: [PATCH] Avoid static members zoom_min_ and zoom_max_

Some compilers cannot use static class members by reference. std::min() and std::max() are passing parameters by const reference.
---
 src/frontends/qt4/GuiView.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/frontends/qt4/GuiView.h b/src/frontends/qt4/GuiView.h
index 0b2b1f3..910df34 100644
--- a/src/frontends/qt4/GuiView.h
+++ b/src/frontends/qt4/GuiView.h
@@ -477,9 +477,9 @@ private:
 	/// from the default zoom pref
 	double zoom_ratio_ = 1.0;
 	/// Minimum zoom percentage
-	static int const zoom_min_ = 10;
+	int const zoom_min_ = 10;
 	/// Maximum zoom percentage
-	static int const zoom_max_ = 1000;
+	int const zoom_max_ = 1000;
 
 	// movability flag of all toolbars
 	bool toolbarsMovable_;
-- 
1.7.1

